src/Beacon.jsx

```diff
@@ -292,7 +290,7 @@ export class Beacon extends Component {
 
     if (appRoot) {
       const oldClone = document.getElementById(TARGET_CLONE_ID);
-      if (!tooltipActive && oldClone) {
+      if (!tooltipActive && oldClone && oldClone.className != 'tour-clone') {
```

Comment:
Prevents the event listener being added multiple times. More a treatment of the symptom than the root cause, which is calling `setState` all over the place and forcing renders. This whole thing needs a rewrite separating it into distinct components for the beacon and the tooltip.